src/services/hooks/use-current-active-block-number.ts

```diff
@@ -12,4 +12,4 @@ const useCurrentActiveBlockNumber = (): UseQueryResult<number, Error> => {
   });
 };
 
-export default useCurrentActiveBlockNumber;
\ No newline at end of file
+export default useCurrentActiveBlockNumber;
```

Comment:
This change came from running `yarn format` while trying to work out while Prettier wasn't running